src/encoded/static/components/static-pages/placeholders/MdSortableTable.js

```diff
@@ -0,0 +1,639 @@
+                } else {
+                    sortIcon = <SortIconDesc />;
+                }
+            } else if (sorting == "asc") {
+                if (iconAsc) {
+                    sortIcon = iconAsc;
+                } else {
+                    sortIcon = <SortIconAsc />;
+                }
+            }
+        }
+
+        return (
+            <div data-field="name" className="markdown-table-column-block" style={{ width }}>
+                <div className="inner" onClick={this.onClick.bind(this)}>
+                    <span className="column-title">{header}</span>
+                    <span className={"column-sort-icon" + (['asc', 'desc'].indexOf(sorting) > -1 ? ' active' : '')}>{sortIcon}</span>
+                </div>
+                <Draggable position={{ x: width, y: 0 }} axis="x" onDrag={this.onDrag} onStop={this.onStop}>
+                    <div className="width-adjuster" />
+                </Draggable>
+            </div>
+        );
+    }
+}
+
+class SortableTableHeader extends React.PureComponent {
+    static propTypes = {
+        columns: PropTypes.array.isRequired,
+        sortings: PropTypes.array.isRequired,
+        onStateChange: PropTypes.func,
+        iconDesc: PropTypes.node,
+        iconAsc: PropTypes.node,
+        iconBoth: PropTypes.node,
+        setHeaderWidths: PropTypes.func.isRequired,
+        headerColumnWidths: PropTypes.array,
+    }
+
+    constructor(props) {
+        super(props);
+
+        this.setHeaderWidths = this.setHeaderWidths.bind(this);
+        this.onAdjusterDrag = this.onAdjusterDrag.bind(this);
+
+        this.state = {
+            'widths': (props.headerColumnWidths && props.headerColumnWidths.slice(0)) || null
+        };
+    }
+
+    onClick(index) {
+        this.props.onStateChange.bind(this)(index);
+    }
+
+    setHeaderWidths(idx, evt, r){
+        const { setHeaderWidths } = this.props;
+        const { widths } = this.state;
+        if (typeof setHeaderWidths !== 'function'){
+            throw new Error('props.setHeaderWidths not a function');
+        }
+        setTimeout(()=> setHeaderWidths(widths.slice(0)), 0);
+    }
+
+    onAdjusterDrag(idx, evt, r){
+        this.setState(({ widths }) => {
+            const nextWidths = widths.slice(0);
+            nextWidths[idx] = r.x;
+            return { 'widths': nextWidths };
+        });
+    }
+
+    getWidthFor(idx){
+        const { widths } = this.state;
+        return Array.isArray(widths) && widths[idx];
+    }
+
+    render() {
+        const { columns, sortings, iconAsc, iconDesc, iconBoth } = this.props;
+        const headers = columns.map(((column, index) => {
+            const sorting = sortings[index];
+            return (
+                <SortableTableHeaderItem {...{ sortable: column.sortable, index, header: column.header, sorting, iconAsc, iconDesc, iconBoth }}
+                    key={index}
+                    onClick={this.onClick.bind(this)}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JSX props should not use .bind() (react/jsx-no-bind)](https://app.codacy.com/app/4dn/fourfront/pullRequest?prid=4682591)